src/Template/Modules/view.twig

```diff
@@ -0,0 +1,31 @@
+{% do _view.assign('title', currentModule.name|humanize) %}
```

Comment:
this file is not used anymore... `src/Template/Pages/Modules/view.twig` is used instead